docs/content/latest/reference/drivers/java/postgres-jdbc-reference.md

```diff
@@ -0,0 +1,310 @@
+
+    ```sh
+    $ openssl verify -CAfile ca.crt -purpose sslclient tlstest.crt
+    ```
+
+4. Convert the client certificate to DER format.
+
+    ```sh
+    $ openssl x509 –in yugabytedb.crt -out yugabytedb.crt.der -outform der
+    ```
+
+5. Convert the client key to pk8 format.
+
+    ```sh
+    $ openssl pkcs8 -topk8 -inform PEM -in yugabytedb.key -outform DER -nocrypt -out yugabytedb.key.pk8
+    ```
+
+Create an `ssl` resource directory in your java application and copy over all the certificates. Update the connection string used by `DriverManager.getConnection` to include the ssl ceritifcates.
+
+Example JDBC URL for connecting to Secure YugabyteDB cluster can be seen below.
+
+```java
+Connection conn = DriverManager.getConnection(jdbc:postgresql://localhost:5433/yugabyte?ssl=true&sslmode=verify-full&sslcert=src/main/resources/ssl/yugabytedb.crt.der&sslkey=src/main/resources/ssl/yugabytedb.key.pk8", "yugabyte", "yugabyte");
+```
+
+| JDBC Params | Description | Default |
+| :---------- | :---------- | :------ |
+| ssl  | Enable SSL JDBC Connection | false
+| ssl-mode |  SSL mode used for JDBC Connection | require
+| sslcert | Client SSL Certificate | yugabytedb.crt
+| sslkey | Client SSL Certificate Key | yugabytedb.key
+
+## Transaction and Isolation Levels
+
+YugabyteDB supports transactions for inserting and querying data from the tables. YugabyteDB supports different [isolation levels](../../../../architecture/transactions/isolation-levels/) for maintaing strong consistency for concurrent data access.
+
+JDBC Driver `java.sql.Connection` interface provides `connection.setAutoCommit()`, `connection.commit()` and `connection.rollback()` methods for enabling transactional access to YugabyteDB Database.
+
+For example
+
+```Java
+Connection conn = DriverManager.getConnection("jdbc:postgresql://localhost:5433/yugabyte","yugabyte", "yugabyte");
+conn.setAutoCommit(false);
+Statment stmt = conn.createStatement();
+try {
+  
+  PreparedStatement pstmt = connection.prepareStatement("INSERT INTO employees (id, name, age, langugage) VALUES (?, ?, ?, ?)");
+  pstmt.setInt(1, 1);
+  pstmt.setString(2, "John");
+  pstmt.setInt(3, 35);
+  pstmt.setString(4, "Java");
+  pstmt.execute();
+
+  conn.commit();
+
+} catch (SQLException e) {
+  System.err.println(e.getMessage());
+}
+
+```
+
+By default PostgreSQL JDBC driver will have `auto-commit` mode enabled which means each SQL statement is treated as a transaction and is automatically committed. If one or more SQL statements are encapsulated in a transaction, auto-commit mode must be disabled by using `setAutoCommit()` method of the `Connection` object.
+
+## Compatibility Matrix
+
+| Driver Version | YugabyteDB Version | Support |
+| :------------- | :----------------- | :------ |
+| 42.2.14 | 2.11 (latest) | full
+| 42.2.14 |  2.8 (stable) | full
+| 42.2.14 | 2.6 | full
+
+## Other Usage Examples
+
+- [JDBC SSL example](../../../../quick-start/build-apps/java/ysql-jdbc-ssl/)
+- [ORM Examples]()
```

Comment:
bad link

Reply:
I will resolve the link.

Reply:
Other Usage examples are not required here. We have removed it for now.